runtime.md

```diff
@@ -15,3 +15,46 @@ Runs a process in a container. Can be invoked several times.
 Not sure we need that from oc cli. Process is killed from the outside.
 
 This event needs to be captured by oc to run onstop event handlers.
+
+## Hooks
+Hooks allow one to run code before/after various lifecycle events of the container. The state of the container is passed to the hooks over stdin, so the hooks could get the information they need to do their work.
+
+The path to the hooks could be absolute or relative and are executed from the host's filesystem. If they are relative, then they are searched for within the PATH environment variable on the host.
+
+The hooks are executed from the container's root filesystem by default, unless overriden in the configuration.
```

Comment:
Didn't undestand this. Like `PATH=rootfs:$PATH` or like we're calling it after `chroot rootfs`? Also how it can be overridden in configuration?